README.md

```diff
@@ -344,7 +344,28 @@ export class UserPreview {
 
 # Forms
 
-> TODO
+### Angular
+```js
+@Component({
```

Comment:
Could you add the necessary imports on top?

Reply:
i am on it.